staging/cse/windows/kubeletfunc.ps1

```diff
@@ -182,7 +182,18 @@ function Get-KubePackage {
         [Parameter(Mandatory = $true)][string]
         $KubeBinariesSASURL
     )
-    Logs-To-Event -TaskName "AKS.WindowsCSE.DownloadKubletBinaries" -TaskMessage "Start to download kubelet binaries and unzip. KubeBinariesPackageSASURL: $global:KubeBinariesPackageSASURL"
+    $mappingFile = [Io.path]::Combine($global:CacheDir, "private-packages\mapping.json")

```

Comment:
Do we have plan to support the package version mapping mechanism for Linux as well? I understand it might be out of the scope of this PR.

Reply:
Shared this with Linux team to do the same thing.